src/Chainweb/Pact/Backend/RelationalCheckpointer.hs

```diff
@@ -124,7 +124,7 @@ doRestore v cid dbenv (Just (bh, hash)) = runBlockEnv dbenv $ do
     setSortedKeys = bsSortedKeys .= pact420Upgrade v bh
 doRestore _ _ dbenv Nothing = runBlockEnv dbenv $ do
     clearPendingTxState
-    withSavepoint DbTransaction $
+    withSavepoint ResetCheckpointer $
```

Comment:
We should double check, that no code elsewhere makes assumptions about this checkpoint name.